### PR TITLE
media-libs/opencolorio: Disable -Werror flag

### PR DESCRIPTION
**Pull request by waebbl**
The src/pyglue/CMakeLists.txt has a global -Werror flag set
for unix systems, which let the compile fail. This patch removes this
flag.

Closes: https://bugs.gentoo.org/666696
Package-Manager: Portage-2.3.62, Repoman-2.3.12
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

### CODE REVIEW

**Comment by waebbl** [comment]
It also adds a bump to EAPI 7, therefore I revbumped the ebuild. Obviously I forgot to add this to the commit comment.
I can do as you ask. Do I need to keyword the 1.1.0-r1 ebuild?

**Comment by a17r** [comment]
No, stable ebuild remains stable, we're just adding a minor build fix.

File: media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild [hunk 1]
```diff
@@ -45,8 +45,11 @@ PATCHES=(
 	"${FILESDIR}/${P}-use-GNUInstallDirs-and-fix-cmake-install-location.patch"
 	"${FILESDIR}/${P}-remove-building-of-bundled-programs.patch"
 	"${FILESDIR}/${P}-yaml-cpp-0.6.patch"
+	"${FILESDIR}/${P}-remove-Werror.patch"
 )
 
+S="${WORKDIR}/OpenColorIO-${PV}"
```
Comment (a17r) [inline media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild]: why is this needed now?

File: media-libs/opencolorio/opencolorio-1.1.0-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Compatibility with Python 3 is declared by upstream, but it is broken in fact, check on bump
```
Comment (a17r) [inline media-libs/opencolorio/opencolorio-1.1.0-r2.ebuild]: is that still relevant?

File: media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild (hunk 1 above)
Comment (waebbl) [reply media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild]: Hmm... looks like it's not. I was probably thinking this is needed because of the different cases in the work subfolder and the name of the build.
I remove it from both ebuilds. Thanks for the hint!

File: media-libs/opencolorio/opencolorio-1.1.0-r2.ebuild (hunk 2 above)
Comment (waebbl) [reply media-libs/opencolorio/opencolorio-1.1.0-r2.ebuild]: I think it's no longer relevant. A basic test in a python shell
`import PyOpenColorIO; PyOpenColorIO.GetCurrentConfig();` works with python-3.6

File: media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild (hunk 1 above)
Comment (a17r) [reply media-libs/opencolorio/opencolorio-1.1.0-r1.ebuild]: Exactly right, -r1 didn't need it because of `vcs-snapshot`.
